src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -24,6 +24,15 @@ public override string ToString()
         }
     }
 
+    public class WalletAccountNameModel : RequestModel
```

Comment:
NACK, wallet name and account name are not both required all the time.
These properties are not inheriting from one another on purpose.